Gruntfile.js

```diff
@@ -255,100 +180,83 @@ module.exports = function (grunt) {
 				]
 			}
 		}
+	},
+	{
+		istanbulIgnoreNext: '/* istanbul ignore next */',
+		staticTestFiles: 'tests/**/*.{html,css}',
+		distDirectory: 'dist',
+		testDirectory: 'test',
+		srcDirectory: 'src',
+		targetDirectory: '<%= devDirectory %>',
+		siteDirectory: '.'
```

Comment:
It required changes in grunt-dojo2, but I have closed that and replaced it with another proposed change.... As soon as that is in and released I'll update this PR.